docs/docker-compose/teuthology/teuthology.sh

```diff
@@ -15,6 +17,7 @@ else
 fi
 export MACHINE_TYPE=${MACHINE_TYPE:-testnode}
 teuthology-suite -v \
+    --teuthology-branch $TEUTHOLOGY_BRANCH \
```

Comment:
Should also let user know in the wait for manual teuth-schedule feature that they need to include this

Reply:
Yeah, when we add that feature we'll have to remember to document that